### ISSUE TITLE
Add robustness CIFAR test sets

### ISSUE THREAD

**Issue by dustinvtran** [google/edward2#96]
For cifar 10.1, add nll / accuracy / ce. For cifar c, add mean nll, mce, and mean ce. similarly cifar p.

**Comment by dustinvtran** [comment]
It looks like adding extra validation sets with the `model.fit` API is fairly involved. We're planning to move to custom training loops anyways, so I'll get started on that first.

**Comment by dustinvtran** [comment]
This issue should be moved to Robustness Metrics' issue tracker once open-sourced.

**Comment by dustinvtran** [comment]
Moved to https://github.com/google/uncertainty-baselines/issues/60.
